var/spack/repos/builtin/packages/perl-gd/package.py

```diff
@@ -10,11 +10,19 @@ class PerlGd(PerlPackage):
     """Interface to Gd Graphics Library"""
 
     homepage = "https://metacpan.org/pod/GD"
-    url = "http://search.cpan.org/CPAN/authors/id/L/LD/LDS/GD-2.53.tar.gz"
+    url = "https://cpan.metacpan.org/authors/id/R/RU/RURBAN/GD-2.77.tar.gz"
```

Comment:
You don't need this property when `url_for_version()` is added.  But I can see keeping it if the older version(s) will be deprecated and there's a plan to remove `url_for_version()`.

Reply:
Let me know if you want to remove it at this time.

Reply:
>  what's 'canonical'?

Depends on context but basically the standard, archetype, or (if form) natural unique representation.